### ISSUE TITLE
Normal type errors cannot be returned by vue-eslint-parser

### ISSUE THREAD

**Issue by ldc2726** [vuejs/vue-eslint-parser#154]
In line 453 it is clear that 'string' cannot be assigned to 'number', but ESLint did not catch this error

![image](https://user-images.githubusercontent.com/34648806/171196751-86ffec07-a51a-4773-b64f-08df66592fbb.png)

``` javascript
module.exports = {
  root: true,
  env: {
    browser: true,
  },
  parser: "vue-eslint-parser",
  extends: [
    'plugin:vue/vue3-essential',
    '@vue/standard',
    'eslint:recommended',
    'plugin:@typescript-eslint/recommended',
    'plugin:@typescript-eslint/eslint-recommended'
  ],
  parserOptions: {
    parser: {
      ts: '@typescript-eslint/parser',
    },
    project: ['./tsconfig.json'],
    extraFileExtensions: ['.vue'],
    // "parser": "@typescript-eslint/parser",
    // "sourceType": "module",
    // project:{
    //   extends:'./tsconfig.json'
    // }
  },
  plugins: [
    "vue",
    "@typescript-eslint",
    // "@typescript-eslint/eslint-plugin",
  ]
}
```

**Comment by ldc2726** [comment]
sorry,Just learned that ESLint does not recommend printing errors to ESLint, but instead uses vue's 'vue-tsc' for detection
